requirements.txt

```diff
@@ -5,3 +5,6 @@ gdal>=3
 #journal      # as of Mar 2022, journal from conda does not support python3.9; since it is included during isce3 installation above, comment this out temporarily.
 ruamel.yaml
 yamale
+packaging
```

Comment:
@yunjunz I am not sure where this dependency comes from. The other two: `backoff` comes from `isce3` (the `stage_dem.py` script requires it), `requests` comes from the Sentinel-1 reader from the script where we download the orbits. Wouldn't it make sense to allocate these two dependencies there?

Reply:
I am not sure why we need `packaging`. Could you @sssangha elaborate?

As for the other two, they should not be added to the `compass` repo, as none of them are not directly used. 

I believe all of us got the `backoff` module import error while installing isce3 through conda. @vbrancat is it possible for you to add it to the isce3 conda recipe?

@sssangha could you please issue a PR to add `requests` to `s1reader` instead?

Reply:
@sssangha piggybacking off @yunjunz comment above, `packaging` should also be included in `s1reader`.

Also did you install isce3 from conda or did you build it from scratch?

IIRC either @rtburns-jpl or @gmgunter do the isce3 conda upload. I'll talk to them on how include `backoff` as a part of the recipe.